i18n/translations/fr/form-builder-responses.json

```diff
@@ -287,19 +287,15 @@
     "title": "Réponses"
   },
   "systemHealth": {
-    "contactSupport": {
-      "title": "Contact Support [FR]",
+    "problemsReported": {
```

Comment:
@anikbrazeau FYI here's some fresh strings.

Reply:
Will add these to my PR :D Thanks for the flag.